ckanext/dcat/converters.py

```diff
@@ -58,13 +67,15 @@ def ckan_to_dcat(package_dict):
 
     dcat_dict = {}
 
-    dcat_dict['title'] = package_dict.get('title')
-    dcat_dict['description'] = package_dict.get('notes')
-    dcat_dict['landingPage'] = package_dict.get('url')
+    dcat_dict['title'] = h.get_translated(package_dict, 'title')
+    dcat_dict['description'] = h.get_translated(package_dict, 'notes')
+    dcat_dict['landingPage'] = h.url_for(package_dict.get('type', 'dataset') \
```

Comment:
The core `url` and the dcat `landingPage` fields of the dataset metadata is not necessarily meant to be the CKAN URL for the dataset page, they can point to another source URL so I would not enforce this here

Reply:
@amercader so the `package_dict.get('url')` would return a possible relative url? But it would be possible for some ckan users to have a package's url to an external url?

Would it be possible to check if the `package_dict.get('url')` is a relative address, and then do the `url_for` as the `url_for` here is to include any language subdomains or sub-directories in use.

Reply:
Resolving for now, new comments regarding the `landingPage` below...might not be able to accomplish my desires though.